lib/pavilion/plugins/commands/status.py

```diff
@@ -255,20 +333,31 @@ def _setup_arguments(self, parser):
             '--history', type=int,
             help="Shows the full status history of a job."
         )
+        parser.add_argument(
```

Comment:
Make this a sub-command, not a regular argument.

`pav status series <args>`. See the show command for an example.